go/general: comments for disapprove (#1223)
exercism/website-copy: automated-comments/go/general/remove_extra_function.md

> One or more extra functions have been created in this solution. They make the code harder to read and can be removed.

Reviewer: ```suggestion
One or more extra functions have been created in this solution. Remove them as they make the code harder to read.
```

Perhaps?

Reviewer 2: This is tricky. What is the context of this comment? Is it an exercise that is supposed to have exactly one function? Or is it an exercise that might be solved with three but definitely not four or five functions?

I think I'd like this to be more imperative and use active voice. Something like:

> Simplify this code to use fewer functions. Unnecessary functions make the code harder to read.

Except that I don't like this either (so I didn't make it a suggestion).

Author: It is supposed to be applied only on a few of the early exercises: `twofer`, `hamming`, maybe `raindrops`, `scrabble` and `isogram` (open for discussion!) where it makes it harder to read when it is split into 2 functions. The condition to trigger this is `len(funcs) > 1`.

Reviewer 2: In that case, can we name the file "single_function" or something explicit like that?

```suggestion
Solve this exercise with a single function. Additional functions in this case make the code harder to read.
```
